README.md

```diff
@@ -113,7 +128,9 @@ MANUAL SECTIONS
     which often include additional sections.
 ```
 
-`time <command>` - gives the time taken for the command to execute. Very useful when you want to find the execution time of your programs.
+`time <command>` - gives the time taken for the command to execute. Very useful when you want to find the execution time of your programs.<br/>
+
+`/usr/bin/time <command>` - to use format flags. Formatting output doesn't work straight away.<br/>
```

Comment:
Remove this... I believe it shouldn't matter if whether you use `time` or `/usr/bin/time`

Reply:
It does not work the normal way when we want to use formatting to get desired output.
Please have a look here : https://asciinema.org/a/0hiwdhObLo2MXkVvABxkkd8kI

Reply:
I see. Change that to " `/usr/bin/time <command>` - To use the "real" time command with all its flags and options. [Reference](https://askubuntu.com/questions/434289/why-doesnt-the-time-command-work-with-any-option) "

And in normal `time <command>` add that "This uses the `time` shell keyword"

Reply:
Ok ! done.